Signal/src/ViewControllers/ColorPickerViewController.swift

```diff
@@ -78,15 +78,18 @@ class ColorPicker: NSObject, ColorPickerViewDelegate {
     private let currentConversationColor: OWSConversationColor
 
     @objc
-    init(currentConversationColor: OWSConversationColor) {
+    init(thread: TSThread) {
+        let colorName = thread.conversationColorName
+        let currentConversationColor = OWSConversationColor.conversationColorOrDefault(colorName: colorName)
         self.currentConversationColor = currentConversationColor
```

Comment:
NBD but why bother with `currentConversationColor` local variable?

Reply:
You're right, we no longer need it. Removed.